habitat_baselines/common/rollout_storage.py

```diff
@@ -4,237 +4,196 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-from collections import defaultdict
+import warnings
 
+import numpy as np
 import torch
 
+from habitat_baselines.common.tensor_dict import TensorDict
+
 
 class RolloutStorage:
     r"""Class for storing rollout information for RL trainers."""
 
     def __init__(
         self,
-        num_steps,
+        numsteps,
         num_envs,
         observation_space,
         action_space,
         recurrent_hidden_state_size,
         num_recurrent_layers=1,
+        is_double_buffered: bool = False,
```

Comment:
Let's add some doc string about double buffering. That's not the easiest abstraction to get from the code without context.

Reply:
It is really just a speed optimization.  It doesn't take any extra memory, it will either speed things up or slow things down.